bke-runtime/gp-bke-runtime-database-postgresql/templates/secrets-added.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-%s" ( include "db.secretName" . ) "added" }}
```

Comment:
"added" find ich nicht schön als Zusatz beim Secret. 
Vielleich etwas wie "user-auth" oder ähnliches?

Reply:
Das ganze Konstrukt mit den zwei Secrets ist nicht schön :-( Im Idealfall gibt es nur ein Object, das man injected. Ich kann es "connect" nennen. Auth ist im anderen Secret drinnen.

Reply:
'connect' finde ich jedenfalls schon schöner. eine andere Möglichkeit wäre, eine ResourceLocker Resource anzulegen, die das Original-secret patched und die fehldenen Werte hinzufügt. 
Bin mir aber nicht sicher, ob das unbedingt viel schöner ist.

Reply:
Man kann das bitnami-Secret auch vorgeben. Dann müsste man das Password selbst generieren und in ein eigens erzeugtes Secret schreiben.